### DISCUSSION TITLE
blacksheep/mvc-project-template/

### DISCUSSION THREAD

**Discussion by giscus** [Neoteroi/BlackSheep-Docs#39] [Q&A]
# blacksheep/mvc-project-template/

The official documentation website of BlackSheep, a fast ASGI web framework for Python asyncio.

https://www.neoteroi.dev/blacksheep/mvc-project-template/

**Comment by OcasoProtal** [comment]
The first screenshot in the section [Server Side Templating](https://www.neoteroi.dev/blacksheep/mvc-project-template/#server-side-templating-views-and-models) is outdated. The file in the screenshot shows the name _hello.html_ while it shoul be _hello.jinja_
